.github/workflows/update-gradle-wrappers-daily.yml

```diff
@@ -0,0 +1,31 @@
+name: Update gradle wrappers (daily)
+
+on:
+  schedule:
+    # daily at 1:30 UTC
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gradle-wrapper:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: setup-java
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 17
```

Comment:
FYI @trask this task fails without installing the JDK. Didn't see any reference to this in the docs for the `update-gradle-wrapper-action`, but updating the wrapper requires running a gradle daemon, which requires java to be installed. Maybe `update-gradle-wrapper-action` installs a lower version of java than is required by opentelemetry-java?

Reply:
interesting, i'm guessing it didn't spit out a useful error to give us any clues?

Reply:
Was testing this on my fork. Here's the build out put without setting up java: https://github.com/jack-berg/opentelemetry-java/actions/runs/5204819774/jobs/9389527088

Reply:
Ya I think we're good to remove